pom.xml

```diff
@@ -20,11 +26,109 @@
         <module>numeraljs</module>
     </modules>
 
+    <properties>
+        <java.level>8</java.level>
+        <node.version>6.4.0</node.version>
+        <npm.version>3.10.3</npm.version>
+    </properties>
+    
     <scm>
         <connection>scm:git:git://github.com/jenkinsci/js-libs.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/js-libs.git</developerConnection>
         <url>https://github.com/jenkinsci/js-libs</url>
         <tag>HEAD</tag>
     </scm>
 
+    <profiles>
+
+        <!--
+            Create a .jar containing all the web resources/assets. Jenkins core can
+            then include the lib, allowing these assets to be loaded via Jenkins
+            core using a URL of the form "<resURL>/assets/<namespace>/<resourcePath>".
+        -->
+        <profile>
+            <id>copy-core-assets</id>
```

Comment:
This all smells like it would be easier wrapped in a Maven plugin... not to mention a faster build.

In fact this looks like a custom lifecycle would be even better

Reply:
Ah will ya stop mess'n. I'll not be doing that :)

Reply:
I do agree it is ugly, but not worth writing a plugin for it.